contracts/LpSugar.vy

```diff
@@ -677,18 +723,31 @@ def _positions(
         else:
           pools_done += 1
 
-        pos_id: uint256 = nfpm.tokenOfOwnerByIndex(_account, pindex)
-        pos: Position = self._cl_position(
-          pos_id,
-          _account,
-          empty(address),
-          empty(address),
-          factory.address,
-          nfpm.address
-        )
+        if pindex < positions_count_nfpm:
+          pos_id: uint256 = nfpm.tokenOfOwnerByIndex(_account, pindex)
+          pos: Position = self._cl_position(
+            pos_id,
+            _account,
+            empty(address),
+            empty(address),
+            factory.address,
+            nfpm.address
+          )
+
+          if pos.lp != empty(address):
+            positions.append(pos)
+        else:
+          pos: Position = self._cl_position(
+            alm_positions_nfpm_ids[pindex - positions_count_nfpm],
+            _account,
+            empty(address),
+            empty(address),
+            factory.address,
+            nfpm.address
```

Comment:
This should probably be `0x...0000` otherwise we won't be able to make a difference between nfpm-managed positions and ALM ones.

Open to other suggestions :thinking: